src/applications/personalization/profile/components/ProfileInfoCard.jsx

```diff
@@ -196,7 +199,7 @@ export const ProfileInfoCard = ({
       {Array.isArray(data) ? (
         <ListOrSections data={data} asList={asList} level={secondaryLevel} />
       ) : (
-        <>{data}</>
+        <div className={classes.firstRow}>{data}</div>
```

Comment:
a non-array based prop should render with the styles of a row, and specifically the first row's style since there will be only one row in this case.